app/controllers/analytics/facilities_controller.rb

```diff
@@ -34,6 +24,10 @@ def set_organization
     @organization = @facility.organization
   end
 
+  def set_cache_key
+    @cache_key = ["view", @facility.analytics_cache_key(@from_time, @to_time)]
```

Comment:
Don't need view here, the view cache do already adds this.

Reply:
Removed `set_cache_key`.